release.ci.jenkins.io.tf

```
@@ -0,0 +1,70 @@
+resource "azurerm_resource_group" "release_ci_controller" {
+  name     = "release-ci"
+  location = "East US 2"
```

```suggestion
  location = var.location
```

Ref. https://github.com/jenkins-infra/azure/blob/main/variables.tf#L1